.github/workflows/backport.yaml

```diff
@@ -0,0 +1,54 @@
+name: Backport
+
+on:
+  pull_request_target:
+    types: [closed]
```

Comment:
Whis is this run when closing the PR, when the actual trigger is on a comment? This event seems unused right now. However, with my suggestion of using labels, this would be the right trigger to use.